src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.php

```diff
@@ -114,6 +115,27 @@
         ;
     }
 
+    if (class_exists(DoctrineDbalAdapter::class)) {
```

Comment:
we prefer removing service in the DI extension as much as possible: config files should remain as declarative as possible

Reply:
I removed the condition. If someone uses `symfony/cache: 5.3` with `symfony/framework-bundle: 5.4` and tries to uses this adapter, he will get an error.

It might be possible to add a tip in the Configuration class.